src/api/config/initializers/wrap_parameters.rb

```diff
@@ -2,7 +2,6 @@
 
 # This file contains settings for ActionController::ParamsWrapper which
 # is enabled by default.
-require 'active_support/core_ext/hash/conversions'
```

Comment:
The comment above still mentions `ActionDispatch::ParamsParser` which is out of the picture now right?

Reply:
It is for `ActionController::ParamsWrapper` and it's still around. See the two `on_load` lines and their blocks below.